src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -128,14 +178,20 @@ private async Task<(Uri? WebURI, Uri? RdpURI)> GetRemoteLaunchURIsAsync(Uri boxU
     }
 
     public ComputeSystemOperations SupportedOperations =>
-        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete;
+        ComputeSystemOperations.Start | ComputeSystemOperations.ShutDown | ComputeSystemOperations.Delete | ComputeSystemOperations.Restart;
 
     public bool IsValid
     {
         get;
         private set;
     }
 
+    public string AlternativeDisplayName => new string("Project : " + ProjectName ?? "Unknown");

```

Comment:
we should add TODOs so we don't forget to add these strings in the resources.resw file

Reply:
Good point.